src/development/sic-calls/history.md

```diff
@@ -1,7 +1,9 @@
 <!-- markdownlint-disable MD024 -->
+<!-- markdownlint-disable MD049 -->
```

Comment:
This disables a lint complaining using `_`  for `*` for italics.
Although I'm okay with the lint, I had to add it to the ignore list since Sophia X user has underscores, so we can't change her `_` in the username for `*` since that would be incorrect.